src/main/java/org/openstreetmap/atlas/geography/atlas/raw/creation/RawAtlasGenerator.java

```diff
@@ -444,8 +455,9 @@ private Atlas rebuildAtlas(final Atlas atlas, final Set<Long> pointsToRemove,
             }
             else
             {
-                logger.debug("Relation {} bean is empty, dropping from Atlas",
-                        relation.getIdentifier());
+                final long relationIdentifier = relation.getIdentifier();
+                logger.debug("Relation {} bean is empty, dropping from Atlas", relationIdentifier);
```

Comment:
bean?

Reply:
RelationBean is an internal PackedAtlas term referring to the relation's collection of members.